aea/crypto/__init__.py

```diff
@@ -19,10 +19,34 @@
 
 """This module contains the crypto modules."""
 
-from aea.crypto.registry import make, register  # noqa
+from aea.crypto.registries import register_crypto, register_ledger_api  # noqa
 
-register(crypto_id="fetchai", entry_point="aea.crypto.fetchai:FetchAICrypto")
+register_crypto(id_="fetchai", entry_point="aea.crypto.fetchai:FetchAICrypto")
+register_crypto(id_="ethereum", entry_point="aea.crypto.ethereum:EthereumCrypto")
+register_crypto(id_="cosmos", entry_point="aea.crypto.cosmos:CosmosCrypto")
 
-register(crypto_id="ethereum", entry_point="aea.crypto.ethereum:EthereumCrypto")
+DEFAULT_FETCHAI_LEDGER_CONFIG = dict(network="testnet")
+register_ledger_api(
+    id_="fetchai",
+    entry_point="aea.crypto.fetchai:FetchAIApi",
+    **DEFAULT_FETCHAI_LEDGER_CONFIG
+)
 
-register(crypto_id="cosmos", entry_point="aea.crypto.cosmos:CosmosCrypto")
+DEFAULT_ETHEREUM_LEDGER_CONFIG = dict(
+    address="https://ropsten.infura.io/v3/f00f7b3ba0e848ddbdc8941c527447fe",
+    gas_price=50,
+)
+register_ledger_api(
+    id_="ethereum",
+    entry_point="aea.crypto.ethereum:EthereumApi",
+    **DEFAULT_ETHEREUM_LEDGER_CONFIG
+)
+
+DEFAULT_COSMOS_LEDGER_CONFIG = dict(
+    address="http://aea-testnet.sandbox.fetch-ai.com:1317",
+)
+register_ledger_api(
+    id_="cosmos",
+    entry_point="aea.crypto.cosmos:CosmosApi",
+    **DEFAULT_COSMOS_LEDGER_CONFIG
```

Comment:
@marcofavorito we still have an issue here that these configs are loaded here; they should be loaded from the "connection.yaml"!

Reply:
we could have the same approach for contracts. No global registry except the cryptos. Are ledger APIs used somewhere else in the framework other than the `LedgerApiConnection`?

Reply:
The idea is that they are only used in the connection. But we do want to allow them to be passed to the simple namespace of an AEA so those developers who do not want to go fully message based can do so.

Reply:
OK, add an issue?

Reply:
Resolved